globus_automate_client/flows_client.py

```diff
@@ -37,11 +37,13 @@
     RUN_STATUS_SCOPE,
 )
 
+AUTHORIZER_CALLBACK_TYPE = Callable[[Tuple[str, str, str]], GlobusAuthorizer]
```

Comment:
Maybe camel case with a starting capital to kinda match types (e.g. class names).